src/master-playlist-controller.js

```diff
@@ -212,7 +212,6 @@ export class MasterPlaylistController extends videojs.EventTarget {
       this.cueTagsTrack_ = this.tech_.addTextTrack('metadata',
         'ad-cues');
       this.cueTagsTrack_.inBandMetadataTrackDispatchType = '';
-      this.tech_.textTracks().addTrack_(this.cueTagsTrack_);
```

Comment:
Shouldn't this have just been changed to `this.tech_.textTracks().addTrack(this.cueTagsTrack_);` rather than removed?